bin/Debugger.Sample/test.js

```diff
@@ -75,6 +75,7 @@ function testConsoleAPIs() {
   console.error('An error');
```

Comment:
Might as well make these consistent and end with a period. 
`console.error("This is an error.");`
`console.warn("This is a warning.");`
`console.info("This is some info.");`